src/index.html

```diff
@@ -15,8 +15,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_head">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="#"
+        />
+      </a>
+      <nav class="navi_box">
+        <ul class="navi_list">
+          <li class="navi_link is-active">
+            <a
+              href="#"
+              class="link is-active"
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```